packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -67,6 +67,12 @@ export interface FieldConfig<TOptions = any> {
   min?: number | null;
   max?: number | null;
 
+  // Interval indicates the expected regular step between values in the series.
+  // When an interval exists, consumers can identify "missing" values when the expected value is not present.
+  // The grafana timeseries visualization will render disconnected values when missing values are found it the time field.
+  // The interval uses the same units as the values.  For time.Time, this is defined in milliseconds.
+  interval?: number | null;
```

Comment:
This is not where we put it in the SDK

Reply:
the interval is on the *field* not the frame:
https://github.com/grafana/grafana-plugin-sdk-go/commit/4ffddbc40722ec2860acb9fc3bafc0eaf3121bc0#diff-b9ef224eea6500d79ee02df546596f8731fd28fcf40256f43e54aa98ebe715c6R48

Reply:
this interface is:
```
export interface FieldConfig<TOptions = any> {
```

Reply:
:man_facepalming: 

yes you are right :)